README.md

```diff
@@ -28,6 +28,9 @@ The following enhancement is added in KoP 2.8.0.
 - Support more clients for Kafka admin
 - Enable advertised listeners, so users can use Envoy Kafka filter directly
 
+## Deprecated features
+The property name of Kafka service listener `listeners` is deprecated. Instead, you can use `kafkaListeners` since KoP 2.8.0.
```

Comment:
confirmed with yunze, it should be `Kafka listener`, instead of `Kafka service listener`.